### DISCUSSION TITLE
Dual Y axis

### DISCUSSION THREAD

**Discussion by matuspu** [hassio-addons/addon-grafana#289] [Q&A]
Dear @frenck is there a way to create timeseries graph with two Y axis in the addon? Grafana docs say you have to click on the series in the graph and select axis in the colour dialogue. In the addon there is no axis tab in the color dialogue. Thank you,  Matus

**Comment by adandl** [comment]
Hello Matuspu!
It took me some time and the tutorials do not seem to be valid for my installation.
You need to go to the override tab and set the unit - this will create a second y-axis:

![image](https://github.com/hassio-addons/addon-grafana/assets/58300252/da828c3d-6a05-4c61-bc52-8085be31db3b)

Hope this helps!
